twitter.go

```diff
@@ -231,6 +231,7 @@ func NewApiError(resp *http.Response) *ApiError {
 	return &ApiError{
```

Comment:
This method looks obsolete by now? Unless it's really meant to be accessed by library users?